dashboard/app/models/pd/enrollment.rb

```diff
@@ -98,6 +98,12 @@ def has_user?
     user_id
   end
 
+  # Returns whether the enrollment owner is deleted. If there is no owner, returns false.
+  # @return [Boolean] Whether the enrollment owner is deleted.
+  def owner_deleted?
```

Comment:
nit: should this be `user_deleted?` since the field is user? Not a big deal. Owner is descriptive enough

Reply:
Keeping it `owner_deleted?` to match other usage, where the owner is through (say) the pd_enrollment.